Solutions/Tomcat/Package/createUiDefinition.json

```diff
@@ -60,41 +60,405 @@
             "name": "dataconnectors1-text",
             "type": "Microsoft.Common.TextBlock",
             "options": {
-              "text": "This Solution installs the data connector for ApacheTomcat. You can get ApacheTomcat custom log data in your Azure Sentinel workspace. Configure and enable this data connector in the Data Connector gallery after this Solution deploys. This data connector creates custom log table(s) Tomcat_CL in your Azure Sentinel / Azure Log Analytics workspace."
+              "text": "The solution installs the data connector ingesting Apache Tomcat internal logging and web application logging will remain independent. After installing the solution, configure and enable this data connector by following guidance in Manage solution view."
             }
           },
           {
             "name": "dataconnectors-parser-text",
             "type": "Microsoft.Common.TextBlock",
             "options": {
-              "text": "The Solution installs a parser that transforms the ingested data into Azure Sentinel normalized format. The normalized format enables better correlation of different types of data from different data sources to drive end-to-end outcomes seamlessly in security monitoring, hunting, incident investigation and response scenarios in Azure Sentinel."
+              "text": "The solution installs a parser that transforms ingested data. The transformed logs can be accessed using the TomcatEvent Kusto Function alias."
             }
           },
           {
-            "name": "dataconnectors-link1",
+            "name": "dataconnectors-link2",
             "type": "Microsoft.Common.TextBlock",
             "options": {
               "link": {
-                "label": "Learn more about normalized format",
-                "uri": "https://docs.microsoft.com/azure/sentinel/normalization-schema"
+                "label": "Learn more about connecting data sources",
+                "uri": "https://docs.microsoft.com/azure/sentinel/connect-data-sources"
               }
             }
+          }
+        ]
+      },
+      {
+        "name": "workbooks",
+        "label": "Workbooks",
+        "subLabel": {
+          "preValidation": "Configure the workbooks",
+          "postValidation": "Done"
+        },
+        "bladeTitle": "Workbooks",
+        "elements": [
+          {
+            "name": "workbooks-text",
+            "type": "Microsoft.Common.TextBlock",
+            "options": {
+              "text": "This solution installs workbook to help you gain insights into the telemetry collected in Microsoft Sentinel. After installing the solution, start using the workbook in Manage solution view."
```

Comment:
Please check if `workbook` has to be updated to `workbook(s)` as per text change document.

![image](https://user-images.githubusercontent.com/107389644/175495805-339d7b5a-7363-4a9b-bc82-6226e0a021c0.png)